var/spack/repos/builtin/packages/tl-expected/package.py

```diff
@@ -15,13 +15,15 @@ class TlExpected(CMakePackage):
 
     maintainers("charmoniumQ")
 
-    license("CC0-1.0")
-
-    # Note that the 1.0.0 has this issue:
-    # https://github.com/TartanLlama/expected/issues/114
-    # But no new patch version has been released,
-    # so I will use the latest commit at the time of writing:
-    version("2022-11-24", commit="b74fecd4448a1a5549402d17ddc51e39faa5020c")
+    license("CC0-1.0", checked_by="wdconinc")
+
+    version("1.1.0", sha256="1db357f46dd2b24447156aaf970c4c40a793ef12a8a9c2ad9e096d9801368df6")
```

Comment:
I can only confirm the version sha256s here if the URL is changed to: `"https://github.com/TartanLlama/expected/archive/refs/tags/v1.0.0.tar.gz"
`.

Reply:
Fixed url.